dev/contributing.rst

```diff
@@ -20,6 +20,50 @@ You must be reachable on the given email address. If you do not wish to use
 your real name for whatever reason, using a nickname or pseudonym is perfectly
 acceptable.
 
+Team Membership
+---------------
+
+Contributing useful commits via pull requests will at some point get you
```

Comment:
I'm not very specific here, as this depends on the frequency of contributions and their quality. There is very little harm to inviting someone into the team, and potentially great benefit in "snaring" someone who were otherwise just doing a drive by commit.